packages/database/schema.prisma

```diff
@@ -87,21 +87,22 @@ model Person {
 }
 
 model Response {
-  id            String   @id @default(cuid())
-  createdAt     DateTime @default(now()) @map(name: "created_at")
-  updatedAt     DateTime @updatedAt @map(name: "updated_at")
-  finished      Boolean  @default(false)
-  survey        Survey   @relation(fields: [surveyId], references: [id], onDelete: Cascade)
-  surveyId      String
-  person        Person?  @relation(fields: [personId], references: [id], onDelete: Cascade)
-  personId      String?
-  responseNotes  ResponseNote[]
-    /// @zod.custom(imports.ZResponseData)
+  id               String         @id @default(cuid())
+  createdAt        DateTime       @default(now()) @map(name: "created_at")
+  updatedAt        DateTime       @updatedAt @map(name: "updated_at")
+  finished         Boolean        @default(false)
+  survey           Survey         @relation(fields: [surveyId], references: [id], onDelete: Cascade)
+  surveyId         String
+  person           Person?        @relation(fields: [personId], references: [id], onDelete: Cascade)
+  personId         String?
+  responseNotes    ResponseNote[]
+  personAttributes Json?
```

Comment:
Can you please add annotations for `prisma-json-types-generator` and `zod-prisma`?
With annotations like
```
/// @zod.custom(imports.ZResponseData)
```
we can export the whole prisma schema into a zod schema and validate outputs based on it (currently in the making). These types can be found in `zod-utils.ts`

and with annotations like
```
/// [ResponseData]
```
we get the types in the prisma typescript exports for `Json` fields. You can find the types in `jsonTypes.ts`